utils/agents/static_RAG_with_coding.py

```diff
@@ -80,10 +93,12 @@ def create_rag_nodes(self):
 
         return workflow
 
-    def build_rag_graph(self, workflow):
+    def build_rag_graph(self, workflow: object) -> object:
 
         # Build graph
 
+        checkpointer: MemorySaver = MemorySaver()
```

Comment:
Is this annotation necessary?

Reply:
Not really, but I like to type most things.  We could do away with it in future if we generalize to more options.